core/src/test/java/com/linecorp/armeria/server/throttling/ThrottlingServiceTest.java

```diff
@@ -46,8 +47,13 @@ protected HttpResponse doGet(ServiceRequestContext ctx, HttpRequest req)
     public ServerRule serverRule = new ServerRule() {
         @Override
         protected void configure(ServerBuilder sb) throws Exception {
-            sb.service("/http-never", SERVICE.decorate(ThrottlingService.newDecorator(never())));
             sb.service("/http-always", SERVICE.decorate(ThrottlingService.newDecorator(always())));
+            sb.service("/http-never", SERVICE.decorate(ThrottlingService.newDecorator(never())));
+            sb.service("/http-never-custom", SERVICE.decorate(
+                    ThrottlingService.newDecorator(ThrottlingStrategy.of((ctx, req) -> completedFuture(false)),
```

Comment:
Why don't we add a test for a custom reject handler?

Reply:
Do you mean custom "accept" handler? If yes, I've added a new test for it.

Reply:
Ah, Thanks!